data/transactions/logic/eval.go

```diff
@@ -282,8 +282,8 @@ type EvalParams struct {
 	SigLedger LedgerForSignature
 	Ledger    LedgerForLogic
 
-	// optional debugger
-	Debugger DebuggerHook
+	// optional tracer
+	Tracer EvalTracer
```

Comment:
I'm worried this might be too similar to the `Trace` field that already exists.

Perhaps I can rename that to `DebugTrace` or similar?